src/server/scripts/Northrend/zone_howling_fjord.cpp

```diff
@@ -324,10 +278,9 @@ enum Razael
     GOSSIP_TEXTID_LYANA2 = 11588
 };
 
-class npc_razael_and_lyana : public CreatureScript
+struct npc_razael_and_lyana : public ScriptedAI
 {
-public:
-    npc_razael_and_lyana() : CreatureScript("npc_razael_and_lyana") { }
+    npc_razael_and_lyana(Creature* creature) : ScriptedAI(creature) { }
 
     bool OnGossipHello(Player* player, Creature* creature) override
```

Comment:
```
/home/runner/work/azerothcore-wotlk/azerothcore-wotlk/src/server/scripts/Northrend/zone_howling_fjord.cpp:285:60: fatal error: only virtual member functions can be marked 'override'
    bool OnGossipHello(Player* player, Creature* creature) override
                                                           ^~~~~~~~
1 error generated.```

Reply:
Yep, that's because we don't have gossip methods inside ScriptedAI, that would need a laaaaarge pr that would break a lot of  npcs and modules, so I strongly recommend to not move scripted gossips npc to the new script system.

Reply:
We need to do it at some point. But for now I'll fix this